package.json

```diff
@@ -10,59 +10,47 @@
 	},
 	"homepage": "https://github.com/utily",
 	"private": false,
-	"main": "dist/index.js",
-	"typings": "dist/index.d.ts",
 	"type": "module",
+	"main": "./dist/cjs/index.js",
+	"module": "./dist/mjs/index.js",
+	"types": "./dist/types/index.d.ts",
+	"exports": {
+		".": {
+			"import": {
+				"default": "./dist/mjs/index.js",
+				"types": "./dist/types/index.d.ts"
+			},
+			"require": {
+				"default": "./dist/cjs/index.js",
+				"types": "./dist/types/index.d.ts"
+			}
+		}
+	},
 	"git": {
 		"tagName": "v${version}"
 	},
 	"github": {
 		"release": true
 	},
-	"jest": {
-		"transform": {
-			"^.+\\.(j|t)sx?$": [
-				"ts-jest",
-				{
-					"tsconfig": "./tsconfig.test.json"
-				}
-			]
-		},
-		"transformIgnorePatterns": [
-			"<rootDir>/node_modules/(?!(cryptly|authly|isoly|gracely|cloudly-http|cloudly-router|cloudly-formdata)/.*)"
-		],
-		"testEnvironment": "node",
-		"testRegex": "((\\.|/)(test|spec))(\\.|\\/.+)(jsx?|tsx?)$",
-		"testPathIgnorePatterns": [
-			"node_modules/",
-			"dist/"
-		],
-		"collectCoverageFrom": [
-			"**/*.{ts,tsx,js,jsx}",
-			"!**/node_modules/**",
-			"!**/dist/**"
-		]
-	},
 	"scripts": {
 		"lint": "eslint --max-warnings=0 '**/*.{ts,tsx}'",
 		"fix": "eslint '**/*.{ts,tsx}' --fix",
-		"build": "tsc -p .",
-		"test": "jest",
+		"build": "rimraf dist/* && tsc -p tsconfig.mjs.json && tsc -p tsconfig.cjs.json",
+		"test": "vitest --run",
+		"coverage": "vitest --coverage",
 		"prepare": "npm run build",
 		"clean": "rimraf dist node_modules coverage"
 	},
 	"devDependencies": {
-		"@types/jest": "^29.5.13",
-		"@types/node": "^22.7.6",
-		"@typescript-eslint/eslint-plugin": "8.10.0",
-		"@typescript-eslint/parser": "8.10.0",
-		"eslint": "^8.20.0",
+		"@types/node": "^22.10.2",
+		"@typescript-eslint/eslint-plugin": "8.18.1",
+		"@typescript-eslint/parser": "8.18.1",
+		"eslint": "^8.56.0",
 		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20231004",
 		"eslint-plugin-simple-import-sort": "^12.1.1",
-		"jest": "^29.7.0",
 		"prettierx": "github:utily/prettierx#utily-20231004",
 		"rimraf": "^6.0.1",
-		"ts-jest": "^29.2.5",
-		"typescript": "5.5.4"
+		"typescript": "5.7.2",
+		"vitest": "^2.1.8"
```

Comment:
`vitest-istanbul` is missing

Reply:
i agree. we should do it explicitly otherwise it does this implicitly under the hood.